cfgov/v1/atomic_elements/organisms.py

```diff
@@ -864,6 +864,9 @@ class MortgageChartBlock(blocks.StructBlock):
     description = blocks.CharBlock(
         required=False,
         help_text='Chart summary for visually impaired users.')
+    note = blocks.CharBlock(
+        required=False,
+        help_text='Note on visualization data.')
```

Comment:
This help text isn't super descriptive but anyone using this block will know what the note field is for so it seems fine.